packages/snapshot-tests/src/formTests.tsx

```diff
@@ -355,6 +355,48 @@ export function formTests(Form: ComponentType<FormProps>, customOptions: FormRen
       const tree = renderer.create(<Form schema={schema} uiSchema={uiSchema} validator={validator} />).toJSON();
       expect(tree).toMatchSnapshot();
     });
+
+    test('checkbox field with description in schema and FieldTemplate', () => {
+      const schema: RJSFSchema = {
+        type: 'boolean',
+        title: 'test',
+        description: 'This is a checkbox description',
+      };
+      const uiSchema: UiSchema = {
+        'ui:widget': 'checkbox',
+        'ui:description': 'This is a field template description',
+      };
+      const tree = renderer.create(<Form schema={schema} uiSchema={uiSchema} validator={validator} />).toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    test('radio widget with description in schema and FieldTemplate', () => {
+      const schema: RJSFSchema = {
+        type: 'boolean',
+        title: 'test',
+        description: 'This is a radio description',
+      };
+      const uiSchema: UiSchema = {
+        'ui:widget': 'radio',
+        'ui:description': 'This is a field template description',
+      };
+      const tree = renderer.create(<Form schema={schema} uiSchema={uiSchema} validator={validator} />).toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    test('select widget with description in schema and FieldTemplate', () => {
+      const schema: RJSFSchema = {
+        type: 'boolean',
+        title: 'test',
+        description: 'This is a select description',
+      };
+      const uiSchema: UiSchema = {
+        'ui:widget': 'select',
+        'ui:description': 'This is a field template description',
```

Comment:
You probably don't need this unless you are testing the UI schema override of the description, which I don't believe you are. Removing this will require an update to the snapshots